lib/experimental/disable-tinymce.php

```diff
@@ -61,14 +61,15 @@ function gutenberg_post_being_edited_requires_classic_block() {
 		return false;
 	}
 
-	// Handle the post editor.
-	if ( ! empty( $_GET['post'] ) && ! empty( $_GET['action'] ) && 'edit' === $_GET['action'] ) {
-		$current_post = get_post( intval( $_GET['post'] ) );
-		if ( ! $current_post || is_wp_error( $current_post ) ) {
-			return false;
-		}
+	// Continue only if we're in the post editor.
+	if ( empty( $_GET['post'] ) || empty( $_GET['action'] ) || 'edit' !== $_GET['action'] ) {
+		return false;
+	}
 
-		$content = $current_post->post_content;
+	// Bail if for some reason the post isn't found.
+	$current_post = get_post( intval( $_GET['post'] ) );
```

Comment:
```suggestion
	$current_post = get_post( absint( $_GET['post'] ) );
```

Reply:
Good call, done in https://github.com/WordPress/gutenberg/pull/52935/commits/b10ca33eac6864bec3634b7543c72ee1a19e510c.